perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -100,15 +101,46 @@
 IODEPTHS_REGEXP = re.compile(r'(\d+)(-(\d+))?$')
 
 
+def GetIODepths(io_depths):
+  """Parse the io_depths parameter.
+
+  Args:
+    io_depths: a string in the format of the --io_depths flag.
+
+  Returns:
+    A list of integers.
+
+  Raises:
+    ValueError if io_depths doesn't follow a format it recognizes.
+  """
+
+  groups = io_depths.split(',')
+  result = []
+
+  for group in groups:
+    match = IODEPTHS_REGEXP.match(group)
+    if match is None:
+      raise ValueError('Invalid io_depths expression %s', io_depths)
+    elif match.group(2) is None:
+      result.append(int(match.group(1)))
+    else:
+      result.extend(range(int(match.group(1)), int(match.group(3)) + 1))
```

Comment:
How do we handle duplicates?
eg. 1-4, 2-5. Do we need to run 2,3,4 twice or sort the final list and only grep uniq values?